src/main/java/step2/domain/Judge.java

```diff
@@ -0,0 +1,47 @@
+package step2.domain;
+
+import step2.dto.JudgeResponse;
+import step2.dto.Lotto;
+import step2.dto.Price;
+import step2.dto.ShopResponse;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Judge {
+
+    private static final int MATCH_CASE = 6 + 1;
+
+    public JudgeResponse calculateResult(ShopResponse shopResponse, Lotto winningLotto) {
+        HashSet<Integer> winningNumbers = new HashSet<>(winningLotto.getNumbers());
+        int[] matchCount = new int[MATCH_CASE];
+        List<Integer> matches = shopResponse.getLottoList()
+                .stream()
+                .map(lotto -> calculateMatchCount(lotto, winningNumbers))
+                .collect(Collectors.toList());
+        matches.forEach(val -> matchCount[val]++);
+        double profitIndex = calculateMoneyWon(matchCount) / shopResponse.getOriginMoney();
+        return new JudgeResponse(matchCount[3], matchCount[4], matchCount[5], matchCount[6], profitIndex);
+    }
+
+    private double calculateMoneyWon(int[] matchCount) {
+        return Price.THREE_MATCH_PRICE * matchCount[3] +
+                Price.FOUR_MATCH_PRICE * matchCount[4] +
+                Price.FIVE_MATCH_PRICE * matchCount[5] +
+                Price.SIX_MATCH_PRICE * matchCount[6];
+    }
+
+    private int calculateMatchCount(Lotto lotto, HashSet<Integer> winningNumbers) {
+        int count = 0;
+        for (int idx = 0; idx < lotto.getNumbers().size(); idx++) {
+            count += oneIfNumberIsInHashSet(lotto.getNumbers().get(idx), winningNumbers);
+        }
+        return count;
+    }
+
+    private int oneIfNumberIsInHashSet(int number, HashSet<Integer> numbers) {
+        if (numbers.contains(number)) return 1;
+        return 0;
```

Comment:
```suggestion
        return numbers.contains(number) ? 1 : 0;
```

취향의 차이겠지만 삼항 연산자를 활용해보는 것은 어떻게 생각하시나요?
삼항 연산자를 사용하지 않더라도 if 문이나 for 문에 대해서는 가능한 brace 를 생략하지 않도록 해주시면 감사하겠습니다.